### ISSUE TITLE
Works in editor, but turns black in browser for WebGL

### ISSUE THREAD

**Issue by justinmelillo** [Ayfel/PrefabLightmapping#46]
Hello! Love this script - it's perfect as we're assetbundling prefabs. 
One thing though, it seems to work when we load the asset bundle in editor, but when we load in our web app, the textures turn black. Any idea what could be causing that? Thanks!

**Comment by Ayfel** [comment]
Are you baking directly from the menu of the Script or just normally?

**Comment by justinmelillo** [comment]
I just ran a test baking directly from the menu of the script...it's just Assets-Bake Prefab Lightmaps, right?

**Comment by Ayfel** [comment]
Yes, can you show me the object with the script that is the prefab? I am looking for the lightmap information in the script. Also did you try making it not a nested prefab?

**Comment by dong6344** [comment]
Did you solve this problem ? 
I have exactly same situation

**Comment by Ayfel** [comment]
I'll need to do a test myself, I will report back once I have checked webgl

**Comment by Ayfel** [comment]
> Yep! The top level prefab "Space" has the script on it: ![image](https://user-images.githubusercontent.com/29789851/137796294-0dffe99c-9a16-4064-9f6c-b89e2c0b868c.png)
> 
> And yes, I un-nested all of those planes (and the point light) to place them directly under Space

Did you try without assetbundles? Not that it should make a difference but just trying to gather info

**Comment by Ayfel** [comment]
I just tested in Unity 2020.3.5 with URP (and without) and in WebGL and it is working fine 
![image](https://user-images.githubusercontent.com/13970424/164487710-a53e9b71-f70f-4a71-9414-91a52c67cbf3.png)

**Comment by Ayfel** [comment]
When using AssetBundles you need to be careful that the project that creates the assetbundle has the same settings for the lightmaps as the one that opens it, specially this 
![image](https://user-images.githubusercontent.com/13970424/164487919-5c3802bc-f896-4e56-ba84-acfa8d72f71b.png)
